eclass/dist-kernel-utils.eclass

```diff
@@ -198,7 +198,19 @@ dist-kernel_compressed_module_cleanup() {
 
 	[[ ${#} -ne 1 ]] && die "${FUNCNAME}: invalid arguments"
 	local path=${1}
-	local preferred=$(dist-kernel_get_module_suffix "${path}/source")
+	local config_path=/usr/src/linux-${KV_FULL}/.config
+
+	local option
+	for option in config source/.config build/.config; do
```

Comment:
Could there be a `source` subdir there, actually?

Reply:
There is in the dist-kernels, and I was under the impression that `make modules_install` did the same, but that turns out to be not the case:

```
ls -l /lib/modules/6.10.3-gentoo-dist
total 5732
lrwxrwxrwx  1 root root      37 Aug  7 11:07 build -> ../../../src/linux-6.10.3-gentoo-dist
lrwxrwxrwx  1 root root      45 Aug  7 11:07 config -> ../../../src/linux-6.10.3-gentoo-dist/.config
drwxr-xr-x 12 root root    4096 Aug  4 22:58 kernel
drwxr-xr-x  2 root root    4096 Aug  7 11:10 misc
-rw-r--r--  1 root root 1327792 Aug  8 15:34 modules.alias
-rw-r--r--  1 root root 1297141 Aug  8 15:34 modules.alias.bin
-rw-r--r--  1 root root   10970 Aug  7 11:06 modules.builtin
-rw-r--r--  1 root root   13523 Aug  8 15:34 modules.builtin.alias.bin
-rw-r--r--  1 root root   13727 Aug  8 15:34 modules.builtin.bin
-rw-r--r--  1 root root   96487 Aug  7 11:06 modules.builtin.modinfo
-rw-r--r--  1 root root  621520 Aug  8 15:34 modules.dep
-rw-r--r--  1 root root  823413 Aug  8 15:34 modules.dep.bin
-rw-r--r--  1 root root     375 Aug  8 15:34 modules.devname
-rw-r--r--  1 root root  185918 Aug  7 11:06 modules.order
-rw-r--r--  1 root root    1453 Aug  8 15:34 modules.softdep
-rw-r--r--  1 root root  650853 Aug  8 15:34 modules.symbols
-rw-r--r--  1 root root  782724 Aug  8 15:34 modules.symbols.bin
lrwxrwxrwx  1 root root      37 Aug  7 11:07 source -> ../../../src/linux-6.10.3-gentoo-dist
lrwxrwxrwx  1 root root      48 Aug  7 11:07 System.map -> ../../../src/linux-6.10.3-gentoo-dist/System.map
drwxr-xr-x  2 root root    4096 Aug  7 11:11 video
lrwxrwxrwx  1 root root      59 Aug  7 11:07 vmlinuz -> ../../../src/linux-6.10.3-gentoo-dist/arch/x86/boot/bzImage
```